_includes/sponsorship/packages.html

```diff
@@ -17,10 +17,11 @@ <h2><span class="label label-success">£2,000</span></h2>
                     <li class="list-group-item">A mention at the beginning and end of the conference</li>
                     <li class="list-group-item">Your logo on the conference website</li>
                     <li class="list-group-item">Mentions on social media channels</li>
-                    <li class="list-group-item">Your name in all emails to participants of SoCraTes UK 2019</li>
+                    <li class="list-group-item">Your name in all emails to participants of SoCraTes UK 2020</li>
                     <li class="list-group-item">Your merchandise available to everyone at the conference</li>
+                    <li class="list-group-item">Support the community: reduce the final ticket price for all attendees</li>
                 </ul>
-                <p><a href="mailto:socratesuk-info@googlegroups.com?subject=Sponsorship%20Premium"
+                <p><a href="mailto:sponsorship-2020@socratesuk.org?subject=Sponsorship%20Premium"
```

Comment:
why the date-dependency here?

Reply:
@jukecraft we thought it might be good in case the email gets picked up by spam bots.
it can of course be changed if we don't think it is necessary